src/noc/containers/noc-affirmation/noc-affirmation.component.html

```diff
@@ -2,16 +2,16 @@
      [ngClass]="{'govuk-form-group--error': validationErrors$ | async}">
   <div class="govuk-form-group">
     <fieldset class="govuk-fieldset" aria-describedby="sign-affirmation">
-      <p>You should tick to 'sign' when:</p>
+      <p>{{"You should tick to 'sign' when:" | rpxTranslate}}</p>
       <ul class="govuk-list govuk-list--bullet">
-        <li>You're satisfied that all these details are accurate and match what is written on the case</li>
-        <li>You have served notice of this change on every party to the case, including the former legal
-          representative (if there was one)
+        <li>{{'Youre satisfied that all these details are accurate and match what is written on the case' | rpxTranslate}}</li>
```

Comment:
Should be 
{{'You\'re satisfied that all these details are accurate and match what is written on the case' | rpxTranslate}}

Reply:
as it contains the single quote inside the text so it will be the problem around the text trans and that's why have to use different format if it contains '

Reply:
I am saying that Youre should be You're